rank/types/elasticsearch.ts

```diff
@@ -2,106 +2,106 @@
+    }
 }
 
 type RecallMetricResponse = {
-  recall: {
-    relevant_docs_retrieved: number
-    docs_retrieved: number
-  }
+    recall: {
+        relevant_docs_retrieved: number
+        docs_retrieved: number
+    }
 }
 
 type MetricResponse = PrecisionMetricResponse | RecallMetricResponse
 
 export type RankDetail = {
-  metric_score: number
-  unrated_docs: unknown[]
-  hits: unknown[]
-  metric_details: MetricResponse
+    metric_score: number
+    unrated_docs: unknown[]
+    hits: unknown[]
+    metric_details: MetricResponse
 }
 
 export type RankEvalResponse<DetailKeys extends string = string> = {
-  metric_score: number
-  details: Record<DetailKeys, RankDetail>
+    metric_score: number
+    details: Record<DetailKeys, RankDetail>
 }
 
 type PrecisionMetric = {
-  precision: {
-    relevant_rating_threshold: number
-    k: number
-  }
+    precision: {
+        relevant_rating_threshold: number
+        k: number
+    }
 }
 
 type RecallMetric = {
-  recall: {
-    relevant_rating_threshold: number
-    k: number
-  }
+    recall: {
+        relevant_rating_threshold: number
+        k: number
+    }
 }
 
 export type Metric = PrecisionMetric | RecallMetric
 
 type RankEvalRequestRating<Rating = 0 | 1 | 2 | 3> = {
-  _index: string
-  _id: string
-  rating: Rating
+    _index: string
+    _id: string
+    rating: Rating
 }
 
 export type RankEvalRequestRequest<Params> = {
-  id: string
-  template_id: string
-  params: Params
-  ratings: RankEvalRequestRating[]
+    id: string
+    template_id: string
+    params: Params
+    ratings: RankEvalRequestRating[]
 }
 
 type RankEvalRequestTemplate = {
-  id: string
-  templates: {
-    inline: unknown
-  }
+    id: string
+    templates: {
+        inline: unknown
+    }
 }
 
 export type RankEvalRequest<TemplateParams> = {
-  templates: RankEvalRequestTemplate[]
-  requests: RankEvalRequestRequest<TemplateParams>[]
-  metric: unknown
+    templates: RankEvalRequestTemplate[]
+    requests: RankEvalRequestRequest<TemplateParams>[]
+    metric: unknown
 }
 
 export type RankEvalResponsWithMeta = RankEvalResponse & {
-  pass: Pass
-  // We've set the passes here as I didn't want to augment the original response.
-  // There's probably a better way to do this, but it'll do for now.
-  passes: Record<string, Pass>
-  queryId: string
-  index: string
-  query: {
-    method: string
-    path: string
-    body: string
-  }
+    pass: Pass
+    // We've set the passes here as I didn't want to augment the original response.
+    // There's probably a better way to do this, but it'll do for now.
+    passes: Record<string, Pass>
+    queryId: string
+    index: string
+    query: {
+        method: string
+        path: string
+        body: string
+    }
 }
 
 export type Hit<DocType = Record<string, any>> = {
-  _id: string
-  _score: string
-  _source: DocType
-  _explanation: Record<string, unknown>
-  highlight: Record<string, string[]>
-  matched_queries: string[]
+    _id: string
```

Comment:
What's with the weird linting?

Reply:
not sure tbh - i was having all sorts of linting issues when i started work on this which i managed to resolve locally, but it looks like they've managed to creep back into this PR